tests/helpers_tests/test_condition.py

```diff
@@ -29,14 +29,18 @@
 semver_equal_2_0_condition_list = [['Android', "2.0", 'custom_attribute', 'semver_eq']]
 semver_equal_2_0_1_beta_condition_list = [['Android', "2.0.1-beta", 'custom_attribute', 'semver_eq']]
 semver_greater_than_2_0_0_condition_list = [['Android', "2.0.0", 'custom_attribute', 'semver_gt']]
+semver_greater_than_2_0_condition_list = [['Android', "2.0", 'custom_attribute', 'semver_gt']]
```

Comment:
Remove all semver conditions from here. Instead declare it as a local variable inside it's respective method.